src/main.rs

```diff
@@ -1,12 +1,30 @@
+#[macro_use] extern crate lazy_static;
+
 use std::io::{self, Read, Write};
 
 const NUM_WINDOWS: usize = 128;
 
+lazy_static! {
+    static ref B64_TABLE: [u8; 64] = {
+        let mut table = [0; 64];
+        for i in 0..26 {
+            table[i] = i as u8 + 'A' as u8;
+        }
+        for i in 0..26 {
+            table[i + 26] = i as u8 + 'a' as u8
+        }
+        for i in 0..10 {
+            table[i + 52] = i as u8 + '0' as u8;
+        }
+        table[62] = '+' as u8;
+        table[63] = '/' as u8;
+        table
+    };
+}
```

Comment:
Does this construct a byte literal at compile time?

Reply:
Unfortunately, [no](https://github.com/rust-lang-nursery/lazy-static.rs#lazy-staticrs) (`const fn` can do that, but it's [unstable](https://doc.rust-lang.org/nightly/unstable-book/language-features/const-fn.html))